README.md

```diff
@@ -45,8 +45,14 @@ This states that `process.env.NODE_ENV` must be set to `development`,
 A default value may be provided:
 
 ```javascript
-const NODE_ENV = envvar.enum('NODE_ENV', ['development', 'staging', 'production'], 'production');
+const NODE_ENV = envvar.oneOf('NODE_ENV', ['development', 'staging', 'production'], 'production');
 ```
 
 This states that `process.env.NODE_ENV` must either be unset (in which case the
 default value is assumed), or set to `development`, `staging`, or `production`.
+
+
+### Migration from `1.x` to `2.x`
+
+  * `enum` was renamed to `oneOf` to avoid shadowing the reserved typescript
+    keyword with the same name.
```

Comment:
This is useful information, but since it is temporal I think it would be better to include it in the [release][1] notes for v2.0.0. Do you agree?


[1]: https://github.com/plaid/envvar/releases